lib/mp-readline/readline.c

```diff
@@ -221,9 +291,43 @@ int readline_process_char(int c) {
             case 'O':
                 rl.escape_seq = ESEQ_ESC_O;
                 break;
+            #if MICROPY_REPL_EMACS_WORDS_MOVE
+                // on stm32, it compiles better when passing parameters to cursr_count_*_word from here.
```

Comment:
Compiling either succeeds or doesn't, so i'm not sure what you mean with 'better' here? Does it produce more optimal code or so? Does that make a runtime difference?

Reply:
By better I meant smaller code size. I didn't actually check what has changed - I don't think it matters since this piece of code runs on user interaction, not in the core of MP, and it's not in the hot path of anything.

I'll clarify the comment

Reply:
Removed since it's incorrect now :/